backend/analytics_server/mhq/api/request_utils.py

```diff
@@ -90,6 +89,11 @@ def coerce_org_repo(repo: Dict[str, str]) -> RawOrgRepo:
         slug=repo.get("slug"),
         idempotency_key=repo.get("idempotency_key"),
         default_branch=repo.get("default_branch"),
+        deployment_type=(
+            TeamReposDeploymentType(repo.get("deployment_type"))
+            if repo.get("deployment_type")
+            else TeamReposDeploymentType.PR_MERGE
+        ),
```

Comment:
What is RawOrgRepo data model? why does it have a TeamRepos property, `deployment type`?
If it is a data model for TeamRepos then why doesn't it have `team_id ` and the name also tell that?

Reply:
Well, it only had org repo data and no team repos data. But it was only being used with team repos based api and services. It would be better to restructure this model. I will do that in the next commit